apollo-runtime/src/appleMain/kotlin/com/apollographql/apollo3/network/http/NSURLSessionHttpEngine.kt

```diff
@@ -35,8 +35,16 @@ fun interface DataTaskFactory {
 
 class NSURLSessionHttpEngine(
     private val dataTaskFactory: DataTaskFactory,
-    private val connectTimeoutMillis: Long = 60_000,
+    private val timeoutMillis: Long = 60_000,
 ) : HttpEngine {
+
+  constructor(
```

Comment:
Nitpick - is there an advantage of adding a secondary constructor vs having a default value for `dataTaskFactory`?

Reply:
Not at all, I'll merge all constructors 👍